addresses/8453.json

```diff
@@ -168,5 +168,10 @@
     "addr": "0x7cb24EFA3fe76650388145b9B0823De6600f1f4c",
     "name": "STRATEGY_MULTICALL",
     "isContract": true
+  },
+  {
+    "addr": "0xb51C52d9E5E41937B0100840b6C3CBA6f7A57A0C",
+    "name": "VIRTUALS_MULTISIG",
+    "isContract": false
```

Comment:
could you add the new mamo multisig for the fees distribution? 

https://moonwellworkspace.slack.com/archives/C08AZFVHLEL/p1751316729786639